chart/permissions-api/templates/deployment-worker.yaml

```diff
@@ -55,13 +55,24 @@ spec:
           env:
             - name: PERMISSIONSAPI_SERVER_LISTEN
               value: ":{{ include "permapi.listenPort" . }}"
+          {{- if eq .Values.config.db.driver "postgres"}}
```

Comment:
Do we need the conditional? Why not just rely on the `with`, similar to how crdb was doing before?

`Values.config.db.driver` should be `Values.config.db.engine`